src/Web/VKHS/Login.hs

```diff
@@ -200,11 +210,14 @@ analyzeResponse (res, jar) = do
           terminate $ LoginFailed $ LoginNoAction
 
 loginRoutine :: (MonadLogin m x s) => Login m x AccessToken
-loginRoutine = initialAction >>= go where
+loginRoutine = (initialAction >>= go) <* saveCookies where
   go a = do
     req <- actionRequest a
     res <- analyzeResponse req
     case res of
       Left a' -> go a'
       Right at -> return at
-
+  saveCookies = do
+    cookies_file <- l_cookies_file <$> toGenericOptions <$> getVKState
```

Comment:
As above, please add an option to skip saving cookies if no filename is given

Reply:
Done!